application/managers/http/manager.py

```diff
@@ -0,0 +1,76 @@
+"""
+Manager for http business logic.
+"""
+
+
+from fastapi import Depends
+
+from constants.events import EventCategory
+from managers.events import EventManager
+from managers.events import get_event_manager
+from managers.kubernetes import K8sManager
+from managers.organizations.manager import OrganizationManager
+from managers.organizations.manager import get_organization_manager
+from models.organization import Organization
+from schemas.events import EventSchema
+from services.helm.facade import HelmService
+import httpx
+
+
+
+class HttpManager:
+    """
+    Manager to manipulate Http requests.
+    """
+    organization_manager: OrganizationManager
+
+    def __init__(self, organization_manager: OrganizationManager, event_manager: EventManager):
+        self.organization_manager = organization_manager
+        self.event_manager = event_manager
+
+
+    ############################################################################
+    # Http Requests
+    ############################################################################
+
+
+    async def http_request(
```

Comment:
I see that method parameters: ` organization context_name component_name dry_run` are unused. I think you can delete them.